src/app/item-popup/DeepsightHarmonizerIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import BungieImage from 'app/dim-ui/BungieImage';
+import { PressTip } from 'app/dim-ui/PressTip';
+import { t } from 'app/i18next-t';
+import { DimItem } from 'app/inventory/item-types';
+import { useD2Definitions } from 'app/manifest/selectors';
+import { DEEPSIGHT_HARMONIZER } from 'app/search/d2-known-values';
+import styles from './DeepSightHarmonizerIcon.m.scss';
+
+export function DeepsightHarmonizerIcon({ item }: { item: DimItem }) {
+  const defs = useD2Definitions()!;
+  const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER)?.displayProperties.icon;
+
+  return (
+    <PressTip tooltip={harmonizableTooltipContent(item)} className={styles.deepsightHarmonizerIcon}>
+      <BungieImage src={harmonizerIcon} />
+    </PressTip>
+  );
+}
+
+function harmonizableTooltipContent(item: DimItem) {
```

Comment:
This, too, should be a component. The difference is that this immediately renders its contents into `DeepsightHarmonizerIcon`'s tree - it's effectively part of that component's render function. Wheras if you make it a component, React will just know "If you need to render this component, you can". Thus, if the tooltip isn't shown, the render function won't be called. The good news is it's only a few characters' difference to make it a component:

```suggestion
function HarmonizableTooltipContent({ item }:{ item: DimItem }) {
```